lib/embed/gpi_embed.c

```diff
@@ -50,6 +50,22 @@ static char *argv[] = { progname };
 
 static PyObject *pEventFn = NULL;
 
+/**
+ * @name    Print the traceback
+ * @brief   Print traceback to stderr
+ *
+ * GILState before calling: N/A
+ *
+ * GILState after calling: N/A
+ *
+ * Print traceback and flush stderr if required (on Windows)
+ */
+void print_traceback(void) {
+  PyErr_Print();
+#if defined(_WIN32)
+  PyRun_SimpleString("import sys; sys.stderr.flush()");
```

Comment:
@LinuxFanatic777 I've never noticed such problem on Linux. Do you try to search in console from which you invoke make? If I can't find error message in transcript tab in Modelsim, I always find it in console.